backend/app/api/endpoints/resource_library.py

```
@@ -285,6 +325,31 @@ def list_my_resource_library_installs(
     )
 
 
+@router.get("/groups/installs", response_model=ResourceLibraryInstallList)
+def list_group_resource_library_installs_batch(
+    group_names: str = Query(..., min_length=1, max_length=20000),
+    resource_type: str | None = Query(
+        default=None, pattern="^(agent|skill|model|shell|retriever)$"
+    ),
+    page: int = Query(default=1, ge=1),
+    limit: int = Query(default=20, ge=1, le=100),
+    db: Session = Depends(get_db),
```

Ruff (0.16.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)